person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(name = 'unknown', age, parent_permission: true) # rubocop:todo Style/OptionalArguments
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :name, :id, :age
```

Comment:
* Since you should have Setters for @name and @age since they should be able to be modified, Kindly define them as `attr_accessors`  only `:id ` is readable